src/components/UserInfo.js

```diff
@@ -15,4 +16,8 @@ export default class UserInfo {
     this._title.textContent = name;
     this._subtitle.textContent = description;
   }
+
+  setUserAvatar(avatar) {
```

Comment:
Great that you've created a method for setting the avatar, since it's controlled by a separate form